modules/era/sql/synth_recipes_era.sql

```diff
@@ -18,453 +18,3476 @@ ALTER TABLE `synth_recipes`
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sphene Earring' AND ID = 22047;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sphene Ring (desynth)' AND ID = 29905;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sphene Ring' AND ID = 22543;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sphene Ring' AND ID = 22579;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Spinel Earring' AND ID = 24028;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Spinel Earring' AND ID = 24064;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Spinel Ring' AND ID = 24538;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Spinel Ring' AND ID = 24570;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Spirit Core' AND ID = 14045;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Spirit Maul' AND ID = 63527;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Spirit Orichalcum Ingot' AND ID = 24070;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Spirit Shell' AND ID = 52503;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Spirit Sword' AND ID = 62541;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Star Earring' AND ID = 24069;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Star Ring' AND ID = 24582;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Staurobow' AND ID = 4560;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Steel Cuisses' AND ID = 12522;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Steel Finger Gauntlets' AND ID = 12506;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Steel Greaves' AND ID = 12532;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Steel Kilij' AND ID = 13011;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Steel Visor' AND ID = 12046;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Steel Walnut Lumber' AND ID = 551;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Steel-splitter' AND ID = 4555;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Stone Bangles' AND ID = 52513;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Stoneskin Torque' AND ID = 23547;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sturdy Slacks' AND ID = 30537;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sturdy Trousers' AND ID = 40506;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sun Earring' AND ID = 23028;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sun Earring' AND ID = 23057;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sun Ring' AND ID = 23523;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sune-Ate' AND ID = 13020;
```

Comment:
Zilart: http://web.archive.org/web/20040624201851/http://ffxi.allakhazam.com/db/item.html?fitem=1506

Reply:
Commenting just to save my place on these, if you find anymore please keep posting them!